tests/commands/test_debug.py

```diff
@@ -73,14 +74,11 @@ def test__locate_error_lines_with_abis_misformatted_line(mock_path, caplog):
     reason="Issue in cairo-lang. "
     "See https://github.com/starkware-libs/cairo-lang/issues/27",
 )
-@patch("nile.utils.debug.subprocess.check_output")
-@patch("nile.utils.debug.GATEWAYS", return_value="123")
-def test_debug_feedback_with_message(
-    mock_gateway, mock_output, output, expected, caplog
-):
+async def test_debug_feedback_with_message(output, expected, caplog):
     logging.getLogger().setLevel(logging.INFO)
-    mock_output.return_value = output
+    with patch("nile.utils.debug.call_cli", new=AsyncMock()) as mock_cli_call:
```

Comment:
```suggestion
    with patch("nile.utils.debug.call_cli") as mock_cli_call:
```
I think `AsyncMock` is the default already?
> If new is omitted, then the target is replaced with an [AsyncMock](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.AsyncMock) if the patched object is an async function or a [MagicMock](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.MagicMock) otherwise.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

Reply:
if so, we should probably update the rest of the tests

Reply:
It's strange, most of the tests follow that approach of _not_ using `new=AsyncMock()`. In the few instances `new=` is still used, it's because the returned mock is a `MagicMock`. I have some guesses, but I'm not definitively sure why. 

Here's an example with `test_debug.py`:

```
async def test_debug_feedback_with_message(output, expected, caplog):
    logging.getLogger().setLevel(logging.INFO)
    with patch("nile.utils.debug.call_cli") as mock_cli_call:
        print("mock type: ", type(mock_cli_call))
```

which returns:

```
tests/commands/test_debug.py::test_debug_feedback_with_message...
mock type:  <class 'unittest.mock.MagicMock'>
```